reach4help/reach4help
# Master -> Staging

## Description
Releasing the latest features into staging environment for QA
Whats's new?

1. Datadog and Firebase 
2. Analytics Integration
3. Improved the Error Messages and Translations on the Personal Data Page
4. Improved Documentation
5. An alternate List view of all requests in the find request page for volunteers.
6. Newer Sign Up Screens
7. Changes in Translation

## Motivation

## Testing Guidelines
Possible New issues:

1. Translations might not work in all pages and might show it as `module.personal_data`  instead of anything sensible for languages other than english (we support spanish, portughese, french) (the chances of this issue showing up is more in the personal data page than in any other page)
2. Make sure that the pages are all translated properly for languages other than english and nothing similar to module.personal_data shows up on any screen for languages other than english (we support spanish, portughese, french) (emphasis on the sign up pages, personal data pages, new request page and the find request page)
3. Ensure sign up is flawless for Google and Facebook
4. Ensure Language change is possible from the landing page and persists across session and isn't based on user's default browser language
5. The process of making an offer for a request works perfectly fine for both the list view and the map view
6. The process of rejecting a request works perfectly fine for both the list view and the map view
7. The process of making a request works perfectly fine
8. The redirections and refresh states are all correct for points 5, 6 and 7 above

## Release Checklist


## Security Checklist

This PR creates, modifies, or deletes:




## Additional Notes
